apps/crc_interactive.py

```diff
@@ -25,7 +25,7 @@ class CrcInteractive(BaseParser):
     min_time = 1  # Minimum limit on requested time in hours
     max_time = 12  # Maximum limit on requested time in hours
 
-    default_time = '1:00'  # Default runtime
+    default_time = '1'  # Default runtime
```

Comment:
This needs to be the same object type as used by the parser argument definition. 

```python
default_time = time(1)
```